wordpress-mobile/WordPress-FluxC-Android
# Create and delete variation Endpoints

Summary
==========
This PR defines endpoints methods to delete or generate an empty variation from a given Product. 

- `generateEmptyVariation` method request the creation of a variation with no value defined and attributes set to `Any` 
- `deleteVariation` completely deletes a variation from the Product.

How to Test
==========

| Enter the Product section inside Woo | Click on Update Product | Interact with generate and delete variations, check the site to verify if changes were applied |
| ------------- | ------------- | ------------- |
| ![Screenshot_20210416_031747](https://user-images.githubusercontent.com/5920403/114980379-c1c79c00-9e62-11eb-891a-1cd30690e48e.png) | ![Screenshot_20210416_031605](https://user-images.githubusercontent.com/5920403/114980464-e1f75b00-9e62-11eb-982c-741f65c2d97d.png) | ![Screenshot_20210416_031531](https://user-images.githubusercontent.com/5920403/114980470-e6bc0f00-9e62-11eb-8ac0-e47ff1747dc3.png) |
